### ISSUE TITLE
headphone problem, Samsung odyssey 2 (NP850XBD-X02US)

### ISSUE THREAD

**Issue by Riei-Joaquim** [pop-os/pop#803]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="19.10"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.10"
VERSION_ID="19.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=eoan
UBUNTU_CODENAME=eoan
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
Internal speakers are recognized and play sound normally. But when plugging the headphone into the headphone and microphone connector, the microphone and headphone are recognized in the sound settings, the microphone works normally but the headphones do not play sound, so you can hear a squeaking volume in the headphones.
NOTE: I tried to install reload and reinstall alsamixer, besides installing other sound controllers / equalizers

**Steps to reproduce (if you know):**
Install Pop OS 19.10 nvidia version on Samsung odyssey 2 (NP850XBD-X02US)

**Expected behavior:**

**Other Notes:**

**Comment by fellipeac2** [comment]
Same problem here. Have you found any solution?

**Comment by Riei-Joaquim** [comment]
No, this problem remains even in version 20.04 LTS, it seems to be a coding problem, because with 100% volume it is possible to listen but with a lot of noise.
